### ISSUE TITLE
Gig work application page

### ISSUE THREAD

**Issue by kkartunov** [topcoder-platform/community-app#4891]
Should be functional and follow design https://marvelapp.com/prototype/f0i4ece/screen/72147213
Exact fields are TBD by the team.

**Comment by SathyaJayabal** [comment]
@kkartunov 
1. Are the below 3 fields mandatory ? (They are mandatory as per the design)
2.Do we have any default vales that are pre selected for these fislds?
<img width="914" alt="Screenshot 2020-09-30 at 2 53 55 PM" src="https://user-images.githubusercontent.com/58783823/94667849-f2177500-032c-11eb-9984-034bc096cb41.png">

3. Blank page displayed on deleting available from date field

**Comment by kkartunov** [comment]
@SathyaJayabal 
3. is now fixed. Please re-test it.
4. Is expected like that to support all kinds of phone formats for numbers. We trust users that they will provide correct numbers so we are able to call them for the gig work opportunity.

**Comment by SathyaJayabal** [comment]
@kkartunov , 3 is verified.

**Comment by kkartunov** [comment]
@SathyaJayabal for 2.&3. 
> They are not mandatory
> They don't have any default options selected

That is what team decision is on that.

**Comment by SathyaJayabal** [comment]
Thanks @kkartunov

**Comment by luizrrodrigues** [comment]
@SathyaJayabal @kkartunov I think we need run QA again in this one, looks like changes made after QA Pass.

changes: 0633ef39f02eb704a2b5b9a3b9ee6804b3670564

**Comment by SathyaJayabal** [comment]
@kkartunov,
1. Some of the mandatory fields dont show the error message when left empty
2. The mandatory field error message doesn't go away after adding data to the field
3. The "Apply to this Job" button is not enabled even after entering all mandatory data

Previously, the "Apply to this Job" button was enabled only after all mandatory data was entered.

**Comment by kkartunov** [comment]
@SathyaJayabal fix for validation issues is deploying. Please, test in ~15 min. Thanks

**Comment by SathyaJayabal** [comment]
@kkartunov, for the CV upload, it allows only .pdf and .docx file when you browse for the files. But you are able to drag and drop other file formats (.zip, mp4, .png etc) and submit your application. What file formats are accepted ?
<img width="1440" alt="Screenshot 2020-10-01 at 1 48 55 PM" src="https://user-images.githubusercontent.com/58783823/94785514-66195200-03ed-11eb-8ee9-07ef52f05a81.png">


Otherwise validation looks good.

**Comment by SathyaJayabal** [comment]
fixed
<img width="1440" alt="Screenshot 2020-10-01 at 2 27 28 PM" src="https://user-images.githubusercontent.com/58783823/94789234-4a647a80-03f2-11eb-816c-121a61ce28fc.png">

**Comment by SathyaJayabal** [comment]
verified on staging (develop)
@kkartunov, validation not happening for mandatory fields (fname, lname, email and city) for logged in users.
[gigs app.mov.zip](https://github.com/topcoder-platform/community-app/files/5311818/gigs.app.mov.zip)

**Comment by SathyaJayabal** [comment]
@kkartunov , as discussed on slack, we are tracking the above validation case in a new issue
https://github.com/topcoder-platform/community-app/issues/5034

**Comment by kkartunov** [comment]
@SathyaJayabal than you. We will address the weird issue as hot-fix in the #5034

**Comment by SathyaJayabal** [comment]
verified on prod
<img width="1440" alt="Screenshot 2020-10-01 at 7 50 24 PM" src="https://user-images.githubusercontent.com/58783823/94821626-66324580-041f-11eb-8501-e49068022055.png">
